templates/chat.mustache

```diff
@@ -45,8 +45,21 @@
 /* eslint-disable promise/no-native */
 (async function() {
   const styleOptions = {
-    hideUploadButton: true
+    hideUploadButton: true,
+    botAvatarImage: "/image/iagora.jpg",
+    botAvatarInitials: 'BF',
+    userAvatarImage: 'https://github.com/compulim.png?size=64',
+    userAvatarInitials: 'WC'
   };
+  const styleSet = window.WebChat.createStyleSet({
+          bubbleBackground: 'rgba(0, 0, 255, .1)',
+          bubbleFromUserBackground: 'rgba(0, 255, 0, .1)'
+        });
+
+        styleSet.textContent = Object.assign({}, styleSet.textContent, {
+          fontFamily: "'Comic Sans MS', 'Arial', sans-serif",
+          fontWeight: 'bold'
+        });
```

Comment:
Same thoughts here; these values should be configurable.
For the `userAvatarImage,` we could use the Moodle user profile image, and for the `userAvatarInitials` - the first letter of the Moodle user's first name and last name.

Reply:
I agree, it's a good idea, I'm going to follow this approach.